tool_quality_checks/scripts.py

```diff
@@ -32,13 +34,17 @@ def run_scripts(settings):
         create_database(settings)
 
     # block with database connection
-    if settings.createdb or settings.import_type or settings.checks:
+    if settings.createdb or settings.import_type or settings.checks or settings.emptydb:
         db = ThreediDatabase(settings)
 
     if settings.createdb:
         logger.info("Initialize the Citybuilder database")
         db.initialize_db_threedi()
 
+    if settings.emptydb:
```

Comment:
Deze had ik nog niet gezien. Settings is meer voor "username" en "db_host".
Hier lijk je het ook te gebruiken als "doe dit"? exporteer/leeg/importeer?

Beetje oppassen dat je zo'n settings object niet misbruikt "omdat het zo'n handig plekje is omdat je dat toch de hele tijd aan alles meegeeft".